contracts/psp22_pallet_wrapper/Cargo.toml

```
@@ -5,16 +5,15 @@ authors = ["Stake Technologies <devops@stake.co.jp>"]
 edition = "2021"
 
 [dependencies]
-ink = { version = "~4.0.0-rc", default-features = false}
-ink_metadata = { version = "~4.0.0-rc", features = ["derive"], optional = true }
+ink = { version = "~4.0.0", default-features = false}
+ink_metadata = { version = "~4.0.0", features = ["derive"], optional = true }
```

```suggestion
ink = { version = "~4.0.0", default-features = false }
```

No need to add `ink_metadata` directly as `ink/std` includes it.
https://github.com/paritytech/ink/blob/bb9cc8cb624c98e4e273b99142c6300a5e399dd3/crates/ink/Cargo.toml#L40